docs/en/observability/whats-new.asciidoc

```diff
@@ -3,21 +3,272 @@
+
+[discrete]
+== Collect logs from AWS Fargate
+
+In this release, we're expanding the AWS Fargate integration with the logs collection.
+Now you can use the {filebeat-ref}/filebeat-module-awsfargate.html[AWS Fargate module] in {filebeat} to
+collect container logs from Amazon ECS on Fargate.
+The module uses the {filebeat} {filebeat-ref}/filebeat-input-aws-cloudwatch.html[`awscloudwatch`]
+input to get log files from one or more log streams in AWS CloudWatch.
+Logs from all containers in Fargate launch type tasks that can be sent to CloudWatch by adding the `awslogs`
+log driver under `logConfiguration` section in the task definition.
+
+[discrete]
+== APM Agents: Improved auto-instrumentation and better support of cloud native services
+
+In 7.13, we've added support for more frameworks when it comes to APM agent auto-instrumentation
+focused on cloud services and frameworks, including:
+
+* Go, Ruby, and Python agents added support for AWS DynamoDB, S3, SNS, and SQS.
+* .NET agent added support for Azure Storage, Queue, and ServiceBus.
+* .NET, Go, and Ruby agents improved support for Azure App Services and now collect additional metadata.
+
+Additional APM agent-specific improvements in 7.13 include:
+
+* Java: Cassandra instrumentation.
+* .NET: MongoDb instrumentation.
+* Node.js: performance improvements.
+* PHP: support for PHP 8, central config support.
+
+[discrete]
+== Native support for OpenTelemetry (GA)
+
+We are pleased to announce that in 7.13, our native support for OpenTelemetry to Elastic Observability is now GA.
+
+OpenTelemetry is emerging as the standardized collection layer for observability data and is designed
+to enable organizations to instrument their apps in an open and vendor-neutral manner.
+Native support for OpenTelemetry protocol (aka OTLP) enables organizations to directly send data
+collected by OpenTelemetry agents to their Elastic deployments.
+Not only does this lower the effort for users to adopt OpenTelemetry into their existing Elastic architecture,
+but it also simplifies the architecture.
+
+Our support for OpenTelemetry is not new. In July 2020, we released the OpenTelemetry Collector exporter for Elastic,
+which acted as a translation layer between OpenTelemetry agents and Elastic APM.
+With the GA of OpenTelemetry protocol support in 7.13, we remove the need to install and manage this extra component.
+Users can now directly send data from their OpenTelemetry agent to the Elastic APM server.
+Using the OpenTelemetry Collector with the OTLP exporter is also supported,
+bringing all the benefits provided by a collector deployed on the edge.
+
+OpenTelemetry integration before v7.12::
+
+[role="screenshot"]
+image::images/7.13-otel-1.png[OpenTelemetry integration before v7.12]
+
+OpenTelemetry integration GA in 7.13 (Experimental in v7.12)::
+
+[role="screenshot"]
+image::images/7.13-otel-2.png[Experimental OpenTelemetry integration in v7.12 (GA in 7.13)]
```

Comment:
I'm really not a fan of taking up so much space to show the *before* image. Is it necessary? Can we just show how it works now? The change is subtle between the two diagrams and will slow readers down.

Reply:
cc @cyrille-leclerc on this one. I agree that they take up a lot of screen real estate. In the Google doc they were side-by-side in a table, but that's unfortunately not reproducible in our doc system.

Reply:
I photoshopped the two images together and added a header in 3c09bd7.

<img width="745" alt="Screen Shot 2021-05-24 at 6 20 27 PM" src="https://user-images.githubusercontent.com/5618806/119425705-b9158180-bcbc-11eb-8110-19ba4963681d.png">